programs/ssl/ssl_server2.c

```diff
@@ -1404,7 +1404,7 @@ int main( int argc, char *argv[] )
 #if defined(MBEDTLS_SSL_CACHE_C)
     mbedtls_ssl_cache_context cache;
 #endif
-#if defined(MBEDTLS_SSL_SESSION_TICKETS)
+#if defined(MBEDTLS_SSL_SESSION_TICKETS) && defined(MBEDTLS_SSL_TICKET_C)
```

Comment:
Ifdef at line 286 - missing `MBEDTLS_SSL_TICKET_C`

Reply:
@mprse I believe it's about the guards around `USAGE_TICKETS` near the top of the file. It's not a major issue (the worst that will happen is that the usage message will mention options that are not actually available in some configs), but not that Andrzej spotted it, we might as well fix it.

Reply:
Yup. I hope it is ok now (fore-pushed last commit).